st2debug/st2debug/cmd/submit_debug_info.py

```diff
@@ -325,7 +392,7 @@ def create_archive(include_logs, include_configs, include_content, include_syste
     return output_file_path
 
 
-def encrypt_archive(archive_file_path, debug=False):
+def encrypt_archive(archive_file_path, debug=False, config_yaml=False):
```

Comment:
Instead of passing in config_yaml should we pass in gpg_key_finger_print=GPG_KEY_FINGERPRINT and have the caller deal with parsing the YAML file.